de.danielnoethen.butt.json

```diff
@@ -0,0 +1,108 @@
+{
+    "app-id": "de.danielnoethen.butt",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "butt",
+    "rename-icon": "butt",
+    "rename-desktop-file": "butt.desktop",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-run/pipewire-0",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.mpris.MediaPlayer2.*",
```

Comment:
why `org.mpris.MediaPlayer2.*` and not `org.mpris.MediaPlayer2.$appname.*`?

Reply:
I'm not really sure what the second one does but my intention is to give the application access to various media players so it can request now playing information.

Reply:
Ok then this is correct.